source/Cute/Commands/Type/TypeDiffCommand.cs

```diff
@@ -1,17 +1,235 @@
 ﻿using Cute.Commands.BaseCommands;
 using Cute.Commands.Login;
 using Cute.Config;
+using Cute.Constants;
 using Cute.Lib.Contentful;
+using Cute.Lib.Exceptions;
 using Cute.Services;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
 using Spectre.Console.Cli;
+using System.Text;
+using Contentful.Core.Models;
+using File = System.IO.File;
+using System.Diagnostics;
+using System.ComponentModel;
+using System.Net.Http;
```

Comment:
This using is redundant. Get CodeMaid and set it up to remove unused usings on save.